modules/nodes-secondary-scheduler-pod-console.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-pod-console_{context}"]
+= Scheduling a pod using the secondary scheduler
+
+To schedule a pod using the secondary scheduler, set the `schedulerName` field in the pod definition.
+
+.Prerequisities
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* The {secondary-scheduler-operator-full} is installed.
+* A secondary scheduler is configured.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Navigate to *Workloads* -> *Pods*.
+. Click *Create Pod*.
+. In the YAML editor, enter the desired pod configuration and add the `schedulerName` field:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  namespace: default
+spec:
+  containers:
+    - name: nginx
+      image: nginx:1.14.2
+      ports:
+        - containerPort: 80
+  schedulerName: secondary-scheduler <1>
+----
+<1> The `schedulerName` must match the name that is defined in the config map when you configured the secondary scheduler.
+
+. Click *Create*.
+
+.Verification
+
+. Log in to the OpenShift CLI.
+. Describe the pod using the following command:
++
+[source,terminal]
+----
+$ oc describe pod nginx -n default
+----
++
+.Example output
+[source,text]
+----
+Name:         nginx
+Namespace:    default
+Priority:     0
+Node:         ci-ln-t0w4r1k-72292-xkqs4-worker-b-xqkxp/10.0.128.3
+...
+Events:
+  Type    Reason          Age   From                 Message
+  ----    ------          ----  ----                 -------
+  Normal  Scheduled       12s   secondary-scheduler  Successfully assigned default/nginx to ci-ln-t0w4r1k-72292-xkqs4-worker-b-xqkxp
+...
+----
+
+. In the events table, find the event with a message similar to `Successfully assigned <namespace>/<pod_name> to <node_name>`.
+. In the "From" column, verify that the event was generated from the secondary scheduler and not the default scheduler.
+
```

Comment:
@bergerhoffer may be worth mentioning that same thing can be seen from the cluster pod logs as well from the openshift-secondary-scheduler-operator namespace

Reply:
Added a note!